sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,291 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 #### Tasks
 
-- [ ] Move the data fetching logic to `pages/_app`
-- [ ] Find a solution for global state handling to have the art pieces available on all pages
-- [ ] Adapt the page `pages/index`: rename the function to `SpotlightPage` and have it render only the `Spotlight` component
-- [ ] Create the page `pages/art-pieces/index` that renders the `ArtPieces` component
-- [ ] Create the component `Navigation` that renders all navigation links
-- [ ] Create the component `Layout` that renders the `Navigation` component
-- [ ] Apply the `Layout` component in `pages/_app`
+- [ ] Create a Navigation component that renders all navigation links.
+- [ ] Render the Navigation component only once, ensuring it appears on all pages.
 
----
+### 4. Art Piece Details Page
 
-### User Story 4: Art Piece Details Page
+When a user clicks on an art piece, they should see all detailed information about it, along with an option to navigate back to the list of art pieces.
 
 #### Value Proposition
 
 **As an** art enthusiast
 
-**I want to** be able to see the full art piece with detail information
+**I want to** view detailed information about an art piece
 
-**so that** I can learn everything about the piece.
+**so that** I can learn everything about it.
 
 #### Acceptance Criteria
 
-- [ ] Clicking an art piece from the list shows the detail page
-- [ ] The art piece image is displayed
-- [ ] The art piece title is displayed
-- [ ] The art piece artist is displayed
-- [ ] The art piece year is displayed
-- [ ] The art piece genre is displayed
-- [ ] A back-button is displayed
-- [ ] Clicking the back-button shows the list view
+- [ ] Clicking on an art piece from the list navigates to the detail page.
+- [ ] The art piece’s image is displayed.
+- [ ] The art piece’s title is displayed.
+- [ ] The art piece’s artist is displayed.
+- [ ] The art piece’s year is displayed.
+- [ ] The art piece’s genre is displayed.
+- [ ] A back button is displayed.
+- [ ] Clicking the back button navigates back to the list view.
 
 #### Tasks
 
-- [ ] Create the component `ArtPieceDetails`
-- [ ] `ArtPieceDetails` props: `image`, `title`, `artist`, `year`, `genre`
-- [ ] Create the page `pages/art-pieces/[slug]` that renders `ArtPieceDetails`
-- [ ] Read the query parameter `slug` from `next/router`
-- [ ] Use the `slug` to find the art piece to display
+- [ ] Create a new component for displaying Art Piece Details.
+- [ ] Create a single dynamic page that renders the Art Piece Details, displaying information for the selected art piece.
+
+### 5. Favorite Page
+
+The Favorite page will display a list of all favorite art pieces. Each art piece will show its image, artist’s name, title, and a favorite button.
+
+Users should be able to save their favorite art pieces both on the Gallery page and the Detail page.
+
+To achieve this, we’ll divide the tasks into two parts:
+
+1. Implement the favorite button where needed.
+2. Create the Favorite page.
+
+### 5.1. Favorite Button
```

Comment:
In my opinion "Favorite Button" is not a descriptive feature name. While it's fine to split the "Bookmark" feature, I think it's important to choose feature-based names for user stories to set a good example.

 For example, a title like "Mark Art Pieces as Favorites" would better represent the feature not the UI element.

```suggestion
### 5.1. Mark Art Pieces as Favorites
```